pkg/pipelines/accesstoken/accesstoken.go

```diff
@@ -32,6 +33,25 @@ func GetAccessToken(gitRepoURL string) (string, error) {
 	return accessToken, nil
 }
 
+//CheckGitAccessToken sets the acceesstoken in the keyring if access-token is present, else it tries to retrieve it from the keyring/env-var
+func CheckGitAccessToken(accessToken, repoURL string) (string, error) {
+	if accessToken != "" {
```

Comment:
This doesn't feel quite right, it sets it if you try to get it with a token, but doesn't actually check it?